yarn.lock

```diff
@@ -2398,7 +2398,12 @@
   dependencies:
     "ajv" "^8.0.0"
 
-"ajv-keywords@^3.4.1", "ajv-keywords@^3.5.2":
+"ajv-keywords@^3.4.1":
```

Comment:
yarn.lock and package.lock should be excluded if no packages are added. Please remove this.